openedx_lti_tool_plugin/tests/test_urls.py

```diff
@@ -2,34 +2,22 @@
 from django.test import TestCase
 from django.urls import resolve, reverse
 
-from openedx_lti_tool_plugin.tests import COURSE_ID, USAGE_KEY
-from openedx_lti_tool_plugin.views import LtiToolJwksView, LtiToolLaunchView, LtiToolLoginView
+from openedx_lti_tool_plugin.views import LtiToolJwksView, LtiToolLoginView
 
 
-class TestUrls(TestCase):
-    """Test openedx_lti_tool_plugin URL configuration."""
+class TestUrlPatterns(TestCase):
+    """Test Django URL configuration."""
 
-    def test_lti_tool_login_url_resolves(self):
-        """Test LtiToolLoginView URL can be resolved."""
+    def test_oidc_login_view_urls(self):
```

Comment:
@kuipumu What is the meaning of oidc?

Reply:
@anfbermudezme This is an abbreviation of "OpenID Connect", this is a reference to the authorization endpoint of the LTI 1.3 launch: https://openid.net/specs/openid-connect-core-1_0.html#AuthorizationEndpoint

I left this change by mistake, I was going to rename the LtiToolLoginView to OIDCLoginView but instead kept the name but this test remained with the new name of the view, I will change it back to `test_lti_tool_login_view_urls`